aiida_raspa/utils/base_parser.py

```diff
@@ -182,6 +228,10 @@ def parse_base_output(output_abs_path, system_name, ncomponents):
                 if parse[0].match(line):
                     parse_block_energy(fobj, result_dict, prop=parse[1])
                     continue  # no need to perform further checks, propperty has been found already
+            for parse in BOX_PROP_LIST:
+                if parse[0].match(line):
+                    parse_box(fobj, result_dict, prop=parse[1])
+                    continue
```

Comment:
I would replace this part for something like following:
```
            for parse in BOX_PROP_LIST:
                if parse[0].match(line):
                    # parse three cell vectors
                    parse_block1(fobj, result_dict, prop='box_ax', value=2, units=3, dev=5)
                    parse_block1(fobj, result_dict, prop='box_by', value=2, units=3, dev=5)
                    parse_block1(fobj, result_dict, prop='box_cz', value=2, units=3, dev=5)
                    # parsee angles between the cell vectors
                    parse_block1(fobj, result_dict, prop='box_alpha', value=3, units=4, dev=6)
                    parse_block1(fobj, result_dict, prop='box_beta', value=3, units=4, dev=6)
                    parse_block1(fobj, result_dict, prop='box_gamma', value=3, units=4, dev=6)
```

This way we don't need to implement a new parsing function and keep the code shorter

Reply:
done.